### PR TITLE
[WIP] October 2016 Meetup - What is GraphQL

### PR DESCRIPTION
**Pull request by hamoore**
Need:
Food Sponsor
Speaker/Talk Bios
Meetup Link

### CODE REVIEW

**Comment by hamoore** [comment]
@ryan-roemer For review, still missing data specified above.

**Comment by ryan-roemer** [comment]
@hamoore -- Looking great so far!

**Comment by hamoore** [comment]
@ryan-roemer ready for final review before posting to meetup/updating with link (and GA link)

File: _posts/2016-10-18-october-2016-what-is-graphql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+As always, you can find all things Seattle Node.js-related at the group website:
+[http://seanode.github.io/](http://seanode.github.io/). And you can reach out to
+us via email at [{{site.author.email}}](mailto:{{site.author.email}}) or Twitter
+at [@{{site.author.twitter}}](https://twitter.com/{{ site.author.twitter }}).
+
+See you soon!
+
+## Agenda
+
```
Comment (ryan-roemer) [inline _posts/2016-10-18-october-2016-what-is-graphql]: Add:

```
* **6:30pm - 7:00pm**: Doors open, hang out with your fellow developers. _Note_ that we do not have a food/beverage sponsor for this event, so grab a bite beforehand!
```

File: _posts/2016-10-18-october-2016-what-is-graphql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+creator of Trails.js (http://trailsjs.io), and father of several cats.
+
+## Venue / Sponsor
+
+**[General Assembly](https://generalassemb.ly/seattle)** has graciously agreed to host the event.
+We encourage you to check out one of the awesome downtown restuarants nearby before or 
+following the event!
+General Assembly is located on the third floor at 1218 3rd Avenue, Seattle, WA, 98101 
+
+[Please RSVP with General Assembly for this event!](xxx)
```
Comment (ryan-roemer) [inline _posts/2016-10-18-october-2016-what-is-graphql]: Is GA going to fill in this link?

Comment (hamoore) [inline _posts/2016-10-18-october-2016-what-is-graphql]: Yes, they will be giving us a link soon. I just pinged her today asking about it.

**Comment by ryan-roemer** [comment]
@hamoore -- I renamed the file to update the date and add the `.md` suffix. Rest looks good.

Once GA gives us event link, let's post meetup event, then finish this PR merging. Thanks!

**Pull request merged**